File: 25_6_2025_json_learn/jsonop.py
import json
print("This is python json operations")

jsonUser="""
{
    "name":"Durgesh",
    "phone":"9839466732",
    "skills":["Java","Python"],
    "address":{
        "city":"LKO",
        "pincode":2455
    }

}
"""

# print(jsonUser)
# print(type(jsonUser))

userDict=json.loads(jsonUser)
print(userDict)
print(type(userDict))
print(userDict['name'])
print(userDict['phone'])
print(userDict['address']['city'])

todo={
    'title':"learn python core for ai",
    'isCompleted':False
}

jsonTodo=json.dumps(todo)
print(type(jsonTodo))
print(jsonTodo)

File: 25_6_2025_json_learn/jsonop.js
console.log("This is my json operations");
const json = `{
    "name":"Durgesh",
    "phone":"9839466732",
    "skills":["Java","Python"],
    "address":{
        "city":"LKO",
        "pincode":2455
    }

}`;

//parse this json to perform operations
const user = JSON.parse(json);

console.log(user.name);
console.log(user.phone);
console.log(user.skills);
console.log(user.address.city);

const todo = {
  title: "learn django",
  isCompleted: true,
};

//convert js object to json string.

const jsonTodo = JSON.stringify(todo);
console.log(jsonTodo);
